function/loader/load.go

```diff
@@ -55,3 +55,40 @@ func Load(ctx context.Context, m PubSubMessage) error {
 	log.Printf("Job created: %s", job.ID())
 	return nil
 }
+
+func LoadStaticAnalysis(ctx context.Context, m PubSubMessage) error {
+	project := os.Getenv("GCP_PROJECT")
+	bucket := os.Getenv("OSSF_MALWARE_STATIC_ANALYSIS_RESULTS")
+
+	bq, err := bigquery.NewClient(ctx, project)
+	if err != nil {
+		log.Panicf("Failed to create BigQuery client: %v", err)
```

Comment:
Not super familiar with the BQ loading parts of this PR, but on the Go side of things:

Is there a reason to panic instead of passing the error up through the return value you have defined? (here and other occurrences in this func)
https://github.com/golang/go/wiki/CodeReviewComments#dont-panic

Reply:
Thanks for the review Spencer. I actually agree with this concept - the code was copied from the existing dynamic analysis `Load()` function above, and only reason I didn't change it was because I wasn't sure what the original reason for including the panic was. Let me find out.

Reply:
Understood, marking as resolved. 

That doc page mentions some performance penalties from cold starts, although it wasn't immediately clear to me if these `panic`s fall into that description. So it may be worth looking into how this compares to their logging library, but that sounds outside the scope of this PR.

Reply:
Actually it looks like it is possible to handle errors in a better way with Cloud Functions, by logging the error as well as returning it.